### ISSUE TITLE
How to override scoresCallbackIntervalMs?

### ISSUE THREAD

**Issue by anoop-sharma** [aws-samples/amazon-chime-react-native-demo#26]
Can anyone help me how to override scoresCallbackIntervalMs in my MeetingObservers.m file? I have been able to override it in Android but not able to figure out for iOS.

**Comment by tianyux-amz** [comment]
Hi @anoop-sharma,

Are you run into any particular issue when overriding `scoresCallbackIntervalMs`?

Try these steps to implement `ActiveSpeakerObserver` in the RN demo app:
1. In [`MeetingObservers.h`](https://github.com/aws-samples/amazon-chime-react-native-demo/blob/master/ios/RNDemo/MeetingObservers.h#L44), make `MeetingObservers` also implement `ActiveSpeakerObserver`.
2. In `MeetingObservers.h`, define the two required properties for `ActiveSpeakerObserver`, e.g.:
```objc
@property (nonatomic, copy) NSString *observerId;
@property (nonatomic) NSInteger scoresCallbackIntervalMs;
```
3. In `MeetingObservers.m`, set `observerId` and `scoresCallbackIntervalMs` in the init method.
4. In [`NativeMobileSDKBridge.m`](https://github.com/aws-samples/amazon-chime-react-native-demo/blob/858c6cee539e373300b93ed9a272ad6efc0048b6/ios/RNDemo/NativeMobileSDKBridge.m#L193), register the `ActiveSpeakerObserver`, e.g.:
```objc
[meetingSession.audioVideo  addActiveSpeakerObserverWithPolicy:[[DefaultActiveSpeakerPolicy alloc] initWithSpeakerWeight:0.9 cutoffThreshold:0.01 takeoverRate:0.2] observer:observer];
```
6. Implement the observer callbacks as you see fit in `MeetingObservers.m`:
```objc
- (void)activeSpeakerDidDetectWithAttendeeInfo:(NSArray<AttendeeInfo *> * _Nonnull)attendeeInfo {}
- (void)activeSpeakerScoreDidChangeWithScores:(NSDictionary<AttendeeInfo *,NSNumber *> *)scores {}
```
Let me know if you have more questions, thanks!

Best,
Tianyu

**Comment by anoop-sharma** [comment]
Dear @tianyux-amz 

Thank you so much for help. Much appreciated.

I faced two issues. One implementing the addActiveSpeakerObserverWithPolicy as the DefaultActiveSpeakerPolicy is not available in my sdk bundle, but when I download the latest one it started working.

Second issue was overriding the scoresCallbackIntervalMs value which I forgot to do in MeetingObservers.m.

But I have been able to resolve all of them.

Much appreciated you help and prompt reply. 

Thanks
